docker/README.md

```diff
@@ -11,18 +11,32 @@ From the root of this repository run:
 image name will be `restic/restic:latest`
 
 ## Run
-
-Set environment variable `RESTIC_REPOSITORY` and map volume to directories and
+Set environment variable `RESTIC_REPOSITORY`, `RESTIC_PASSWORD` and map volume to directories and
 files like:
 
 ```
-docker run --rm --hostname my-host -ti \
-    -v $HOME/.restic/passfile:/pass \
+docker run --rm --hostname my-host \
     -v $HOME/importantdirectory:/data \
     -e RESTIC_REPOSITORY=rest:https://user:pass@hostname/ \
-    restic/restic -p /pass backup /data
+    -e RESTIC_PASSWORD=/run/secrets/restic_password
+    restic/restic
 ```
 
 Restic relies on the hostname for various operations. Make sure to set a static
 hostname using `--hostname` when creating a Docker container, otherwise Docker
 will assign a random hostname each time.
+
+## Advanced
+Optionally you can change the source directory by setting `RESTIC_DATA` to an other path, 
+defaults to /data.  
+Or you can enable a [retention policy](https://restic.readthedocs.io/en/stable/060_forget.html#removing-snapshots-according-to-a-policy) 
+by setting `RESTIC_FORGET`.
+For other possible environment variables check: [Environment Variables](https://restic.readthedocs.io/en/stable/040_backup.html#environment-variables)
+
+By removing the *--rm* tag, the container would not destroy itself after it is finished, 
+only exits, and you can schedule your backups just by starting the container with cron like this:
+```cron
+0 0 * * * docker start restic
```

Comment:
It feels rather weird to stop and restart a docker container later on. AFAIK it's much more canonical to start new fresh containers each time. This will require storing the cache on a separate volume for optimal performance though. Always reusing the same container also complicates upgrading restic. You'd have to manually delete the container in this case.

Besides that, this construction prevents setups that for example run `forget` and `prune` only once a week. Or similarly run `check` on a weekly basis.